FE/todo-app/src/components/createGroupForm/CreateGroupForm.tsx

```diff
@@ -32,8 +32,8 @@ export const CreateGroupForm: FC<Props> = ({ open, closeModal }) => {
         to_dos: [],
       },
     };
-    createGroup(dataInput);
-
+    await Promise.all([createGroup(dataInput).unwrap()]);
+    methods.setValue("title", "");
     closeModal();
```

Comment:
tohle máš celé nějaké divné ne?
Proč je to v promise.all?? a neměl by jsi ten unwrap ukožit do proměné a čekat na nějaký stav a až podle toho stavu zavřít modál a setnout value??